Lombiq.Tests.UI/Services/UITestContext.cs

```diff
@@ -89,6 +89,13 @@ public class UITestContext
     /// </summary>
     public string TenantName { get; set; } = "Default";
 
+    /// <summary>
+    /// Gets or sets the current admin prefix if a custom one is needed. When creating UI tests on a site that
+    /// has custom admin prefix this value should be set in the UI test. After that navigation methods will be able
+    /// to catch up to the custom url.
```

Comment:
Break to 120 characters.

```suggestion
    /// Gets or sets the current Orchard Core admin prefix. When running UI tests on a site that
    /// uses a custom admin prefix, this value should be set in the test. After that, navigation methods will be able
    /// to use the custom URL.
```